packet_forwarder/global_conf.json.sx1250.AS923.AS1

```diff
@@ -14,7 +14,11 @@
       "enable": true,
       "type": "SX1250",
       "freq": 923000000,
+<<<<<<< HEAD:packet_forwarder/global_conf.json.sx1250.AS923.AS1
```

Comment:
You accidentially checked in a merge conflict

Reply:
Sorry for the YOLO merge and for the lack of follow up! My github spam in my inbox is untenable and I appear to be missing things. Anyway, I think this is okay now pending changes in nextgate for region selection.

Reply:
Sorry I spoke too soon. Still fixing